templates/common/infra/bicep/core/cdn/cdn-endpoint.bicep

```diff
@@ -0,0 +1,49 @@
+param name string
```

Comment:
I'd like to put this in a folder named "networking" instead of cdn.

Reply:
Sure, moved to networking